CHANGES.md

```diff
@@ -205,6 +205,8 @@ var viewer = new Viewer("cesiumContainer", {
 
 ##### Fixes :wrench:
 
+<<<<<<< HEAD
```

Comment:
Looks like there were some merge issues with this file.

Reply:
woops! Fixed :)